applications/FluidDynamicsApplication/custom_elements/vms_adjoint_element.cpp

```diff
@@ -6,141 +20,135 @@ namespace Kratos {
-            this->GetGeometry()[iNode].GetDof(ADJOINT_FLUID_VECTOR_1_X).EquationId();
-        rResult[LocalIndex++] =
-            this->GetGeometry()[iNode].GetDof(ADJOINT_FLUID_VECTOR_1_Y).EquationId();
-        rResult[LocalIndex++] =
-            this->GetGeometry()[iNode].GetDof(ADJOINT_FLUID_SCALAR_1).EquationId();
+    IndexType local_index = 0;
+    for (IndexType i_node = 0; i_node < TNumNodes; ++i_node) {
+        const auto& r_node = this->GetGeometry()[i_node];
+        rResult[local_index++] = r_node.GetDof(ADJOINT_FLUID_VECTOR_1_X).EquationId();
+        rResult[local_index++] = r_node.GetDof(ADJOINT_FLUID_VECTOR_1_Y).EquationId();
+        rResult[local_index++] = r_node.GetDof(ADJOINT_FLUID_SCALAR_1).EquationId();
     }
 }
 
 template <>
-void VMSAdjointElement<3>::EquationIdArray(EquationIdArrayType& rResult,
-                                           const ProcessInfo& rCurrentProcessInfo) const 
+void VMSAdjointElement<3>::EquationIdArray(
+    EquationIdArrayType& rResult,
+    const ProcessInfo& rProcessInfo) const
 {
-    IndexType LocalIndex = 0;
-    for (IndexType iNode = 0; iNode < TNumNodes; ++iNode)
-    {
-        rResult[LocalIndex++] =
-            this->GetGeometry()[iNode].GetDof(ADJOINT_FLUID_VECTOR_1_X).EquationId();
-        rResult[LocalIndex++] =
-            this->GetGeometry()[iNode].GetDof(ADJOINT_FLUID_VECTOR_1_Y).EquationId();
-        rResult[LocalIndex++] =
-            this->GetGeometry()[iNode].GetDof(ADJOINT_FLUID_VECTOR_1_Z).EquationId();
-        rResult[LocalIndex++] =
-            this->GetGeometry()[iNode].GetDof(ADJOINT_FLUID_SCALAR_1).EquationId();
+    IndexType local_index = 0;
+    for (IndexType i_node = 0; i_node < TNumNodes; ++i_node) {
+        const auto& r_node = this->GetGeometry()[i_node];
+        rResult[local_index++] = r_node.GetDof(ADJOINT_FLUID_VECTOR_1_X).EquationId();
+        rResult[local_index++] = r_node.GetDof(ADJOINT_FLUID_VECTOR_1_Y).EquationId();
+        rResult[local_index++] = r_node.GetDof(ADJOINT_FLUID_VECTOR_1_Z).EquationId();
+        rResult[local_index++] = r_node.GetDof(ADJOINT_FLUID_SCALAR_1).EquationId();
     }
 }
 
 template<>
-double VMSAdjointElement<2>::CalculateElementSize(const double Area)
+double VMSAdjointElement<2>::CalculateElementSize(
+    const double Area) const
 {
-  // Diameter of a circle of given Area.
-  return 1.128379167 * std::sqrt(Area);
+    // Diameter of a circle of given Area.
+    return 1.128379167 * std::sqrt(Area);
 }
 
 template<>
-double VMSAdjointElement<3>::CalculateElementSize(const double Volume)
+double VMSAdjointElement<3>::CalculateElementSize(
+    const double Volume) const
 {
-  // Diameter of the sphere circumscribed to a regular tetrahedron
-  // with the same volume.
-  return 0.60046878 * std::pow(Volume,0.333333333333333333333);
+    // Diameter of the sphere circumscribed to a regular tetrahedron
+    // with the same volume.
+    return 0.60046878 * std::pow(Volume, 0.333333333333333333333);
 }
```

Comment:
Why not taking these from the `ElementSizeCalculator`?. If I'm not wrong, this way of calculating the element size is not implemented but we could take the chance to add it (just suggesting).

Reply:
Wouldn't that be an overkill? `ElementSizeCalculator` calculates everything from scratch, whereas here we use already calculated Area and Volume from GaussPointCalculations?

Reply:
Well, it would imply to calculate the area/volume again. But considering that this can be hardcoded according to the element type (it is templated) I wouldn't say it is an overkill.

Reply:
In any case, we can leave it for a future PR as this is only clean up.